arm-servicefabric/2016-09-01/examples/ClusterPutOperation_exmaple.json

```diff
@@ -0,0 +1,323 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "resRg",
+    "clusterName": "myCluster",
+    "api-version": "2016-09-01",
+    "ClusterUpdateParameters": {
+      "properties": {
+        "availableClusterVersions": [
+          {
+            "codeVersion": "5.1.158.9590",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.2.0.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.2.207.9590",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.3.121.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.3.311.9590",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.30.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.35.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.38.100",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.118.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.132.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
+            "environment": "Windows"
+          },
+          {
+            "codeVersion": "5.4.145.9494",
+            "supportExpiryUtc": "9999-12-31T23:59:59.9999999",
```

Comment:
Log a bug against your service to fix the datetime in and out to be in ISO 8601. This dateTime is missing timezone.